packages/user-interface/src/components/Notification.module.scss

```diff
@@ -0,0 +1,8 @@
+.nl-portal-notification {
```

Comment:
Deze hoeft eigenlijk helemaal geen specifieke naam te hebben aangezien dit een module is en alleen leeft in het component.

Reply:
Nee oké, maar dit is ook geen probleem toch?